Automattic/wp-calypso
# Atomic: Add blocked message to blocked eligibility

#### Changes proposed in this Pull Request

This add a new eligibility check message for sites that have been blocked from installing plugins and themes.

#### Testing instructions
1. Setup a site per the test plan in D26465-code
2. Try transferring the site by installing a plugin or uploading a theme.
3. You should see the ineligibility screen :
![Screen Shot 2019-04-03 at 4 01 40 PM](https://user-images.githubusercontent.com/744755/55520584-dc3ecc00-5631-11e9-9a0d-c61a91dbef7c.png)
